frescobaldi_app/open_file_at_cursor.py

```diff
@@ -33,7 +33,7 @@
 import browseriface
 
 # regular expression for finding \include expressions
-incl_regex = re.compile('(\\\\include\s*\")(.*)(\")')
+incl_regex = re.compile(r'(\\include\s*")(.*)(")')
```

Comment:
suggestion: When you are changing the regex you might aswell change it to `r'(\\include\s*")([^"]*)(")'`. Which is more precise.